### ISSUE TITLE
New trait data entry can not be added to BETY

### ISSUE THREAD

**Issue by lovelilyinfall** [PecanProject/bety#221]
I recently have some problems adding new trait data to BETY through the web interface. After I enter the values for the new trait and click “create”, I get a confirmation page saying “Trait was successfully created”. But below that line it says “404 Sorry no Trait/Yield data here ”. When I try to search the trait data I have just entered using species name “Psychotria berteriana”, it shows no entries found. I really need to get these data entered to the database because I need them for the meta-analyses for my ED runs. I have added some screenshots to show what happened exactly. Thanks for your help!

![add new trait](https://cloud.githubusercontent.com/assets/5421381/6094909/78d05968-af0f-11e4-9cfa-e70b058145bd.JPG)

![added triat cannot be found](https://cloud.githubusercontent.com/assets/5421381/6094911/8017e542-af0f-11e4-8028-f271257a8725.JPG)

![no entries found](https://cloud.githubusercontent.com/assets/5421381/6094914/8f020f60-af0f-11e4-84d5-e14d81908297.JPG)

**Comment by dlebauer** [comment]
I've changed your access_level from 3 to 2. See if this works now.

I've marked this as a bug because you should be able to enter data with access level 3.

@gsrohde this is username `feng22` who has `page_access_level = 2` (manager); permission to add records should be available with `page_access_level` >=3`, and I do not think it should depend on the users `access_level`,

**Comment by lovelilyinfall** [comment]
@dlebauer Yes, it is working. Thanks!

**Comment by gsrohde** [comment]
@dlebauer 

The traits do get added, even when the user's access level is 3.  The problem is that the default access_level for new traits is 2, and users aren't allowed to view rows that have a more restrictive access_level than the access_level they are logged in with.

Perhaps the solution is to make an exception if the user_id on the row matches their own user_id.  This way a user will always be able to see and edit the rows that they create.  Maybe things once worked this way, since I don't remember this having been a problem before.

**Comment by dlebauer** [comment]
> exception if the user_id on the row matches their own user_id

Indeed, this was the intent of the design - so users can always edit data that they created.
